### PR TITLE
Fix/nextjs support

### PR DESCRIPTION
**Pull request by micwallace**
Hey @nicktaras 

This branch adds these things: 

1. Removed CSS import within token negotiator. Many frameworks, including next.js don't allow CSS imports from within node modules. Examples have been updated to import CSS on wd-example branch. 
2. On chain authentication challenge signing and and address signature check. 
3. Fab overlay displays loader & message when authenticating in order to improve UX.

### CODE REVIEW

**Comment by micwallace** [comment]
@jot2re Can you have a quick look at this. It's getting UN from the same place as off-chain authentication (crypto verify server) and getting the user to sign a message with it. It then verifies the signature on the frontend. The signature can then be sent to Webster's backend for verification, along with NFT ownership check: 
https://github.com/TokenScript/token-negotiator/blob/14c024507480f26adbe077c61965bdf668d25173/src/client/index.ts#L336

Websters server should get the current UN from the crypto verify service independently, otherwise the frontend can specify any signed challenge.

**Comment by jot2re** [comment]
The code seems reasonable to me for the flow where a third party webserver wants to verify that the user owns a specific token (that exists on the blockchain).

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tokenscript/token-negotiator",
-  "version": "1.0.18-alpha",
+  "version": "1.0.19-alpha-wd",
```
Comment (nicktaras) [inline package.json:3]: Can we remove the -wd from the version. Let's move to using Semantic versioning.

https://docs.npmjs.com/about-semantic-versioning (applying these rules to future updates)

File: src/client/index.ts [hunk 2]
```diff
@@ -14,7 +14,6 @@ import {
 } from "../tokenLookup";
 import OnChainTokenModule from "./../onChainTokenModule";
 import Web3WalletProvider from "./../utils/Web3WalletProvider";
-import "./../theme/style.css";
```
Comment (nicktaras) [inline src/client/index.ts:17]: Thanks for providing more detail around the issues from the Next JS forum about this (https://github.com/vercel/next.js/issues/19936) - the reasoning makes sense, even though it incurs more work for web developers.

To implement this change can you update the ReadMe with steps for the developers to install the CSS alongside the core lib.

File: src/client/index.ts [hunk 3]
```diff
@@ -25,6 +24,7 @@ interface NegotiationInterface {
     filters: {};
   };
   onChainKeys?: { [apiName: string]: string };
+  ipfsBaseUrl?: string
```
Comment (nicktaras) [inline src/client/index.ts:27]: Good idea :) Especially if we find people want to offload requests onto their servers.

File: package.json (hunk 1 above)
Comment (micwallace) [reply package.json:3]: Yep this was only so Feng can build the brand connector demo.

File: src/client/index.ts (hunk 2 above)
Comment (micwallace) [reply src/client/index.ts:17]: @nicktaras yes, readme now updated.

File: src/client/index.ts (hunk 3 above)
Comment (micwallace) [reply src/client/index.ts:27]: Yeah it make things a lot faster when using Moralis/Alchemy, which returns raw metadata that often includes ipfs links using public gateways. This is different from Opensea which returns a cached CDN URL.

**Pull request merged**